platform/linux-generic/odp_schedule_basic.c

```diff
@@ -221,11 +226,41 @@ static __thread sched_local_t sched_local;
 /* Function prototypes */
 static inline void schedule_release_context(void);
 
+static int read_config_file(sched_global_t *sched)
+{
+	const char *str;
+	int val = 0;
+
+	ODP_PRINT("Scheduler config:\n");
+
+	str = "sched_basic.prio_spread";
+	if (!_odp_libconfig_lookup_int(str, &val)) {
+		ODP_ERR("Config option '%s' not found.\n", str);
+		return -1;
+	}
+
+	if (val > MAX_SPREAD || val < MIN_SPREAD) {
+		ODP_ERR("Bad value %s = %u\n", str, val);
```

Comment:
Saturation is better than an error here.

Reply:
Error is better. Saturation would cause e.g. a file with garbage values to set everything silently to max/min - which is not typically what user expects to happen. Exact values from config file are either  used or not - otherwise user can not be sure what config he's actually running.